clientapi/routing/createroom.go

```diff
@@ -212,6 +206,107 @@ func createRoom(
 		return jsonerror.InternalServerError()
 	}
 
+	createContent := map[string]interface{}{}
+	if len(r.CreationContent) > 0 {
```

Comment:
Why not just have this inlined in the struct as it was before?

Reply:
There seems to be sytests for being able to put arbitrary fields into the create event. If we use the struct then they get stripped marshalling in/out of the struct.